src/main/java/seedu/address/storage/XmlAdaptedVisitor.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.storage;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import seedu.address.model.visitor.Visitor;
+
+/**
+ * JAXB-friendly adapted version of the Visitor class.
+ * @author GAO JIAXIN666
+ */
+@XmlRootElement
+public class XmlAdaptedVisitor {
+    @XmlElement(required = true)
+    private String visitor;
+
+    /**
+     * Constructs an XmlAdaptedDiagnosis.
```

Comment:
Clearly inaccurate comment.
Please (at the very least) edit your comments to be accurate if you're going to copy-paste code.